sparkbox/safe-focus
# chore: update standard-version to 9.0.0

This PR address security vulnerabilities within the `standard-version` package and updates it to 9.0.0.

## Validation
1. Checkout `master`.
1. Run `rm -rf node_modules && npm i`.
1. Run `npm audit`
- [ ] Confirm 4 vulnerabilities require semver-major dependency updates.
1. Checkout `chore--security-updates`.
1. Run `rm -rf node_modules && npm i`.
1. Run `npm run test`
1. Run `npm audit`
- [ ] Confirm 0 vulnerabilities are listed.
